### DISCUSSION TITLE
Easy write DateTime

### DISCUSSION THREAD

**Discussion by ebattulga** [dotnet/csharplang#6829] [Language Ideas] [answered]
I would like to write DateTime in just like that 
`DateTime dt='2022.12.22'dt;`
instead of `DateTime dt=new DateTime(2022,12,22);`
Sometimes work with multiple static datetime insertion, it is very annoying. 
When i write this `'2022.12.22'dt` compiler lowered to it `new DateTime(2022,12,22)`
'2022/12/22'dt => new DateTime(2022,12,22);
'2022/12/22 10:40'dt => new DateTime(2022,12,22,10,40,0);

`TimeSpan ts='2022.12.22'dt - '1984.08.30'dt ;`
etc

Single or Double quote doesn't matter.

**Comment by ufcpp** [comment, marked as answer]
https://github.com/dotnet/csharplang/discussions/1205
https://github.com/dotnet/csharplang/discussions/1969
